week-1-ts-ecosystem/w01e03-object-oriented-structure/src/types/product.ts

```diff
@@ -0,0 +1,21 @@
+export interface Product<Type extends ProductType> {
```

Comment:
Ten inferfejs jest "słoniem w składzie porcelany"

Ponieważ Twój model danych i jego kształt posiada:
`product.model.ts`

To tam mamy klasę `Product` - więc nie potrzeba nam tego interface'u.